portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/wysiwyg/usecase/WYSIWYGUsecase.testcase

```diff
@@ -263,6 +263,82 @@ definition {
 		AssertElementPresent(locator1 = "//img[contains(@src,'Document_1.jpg')]");
 	}
 
+	@priority = "4"
+	test PreviewSourceContent {
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
```

Comment:
I wonder if the `&amp;` is necessary... I see a few other tests with just `&` working.

Reply:
`&amp;` is not necessary. 
This category name was updated in all tests in PR: https://github.com/brianchandotcom/liferay-portal/pull/68652.

Do you think we should change it to "Content & Date in all tests?

Reply:
@david-gutierrez-mesa what do you think, since this is Product Menu? I'm recommending that we replace `&amp;` with just `&`.

Reply:
@zhangyan0701 , we can do this in a separate ticket if approved with David.